Packs/ApacheWebServer/ModelingRules/ApacheWebServerModelingRules_1_3/ApacheWebServerModelingRules_1_3_schema.json

```diff
@@ -0,0 +1,14 @@
+{
+    "Apache_httpd_error": {
+      "_raw_log": {
+        "type": "string",
+        "is_array": false
+      }
+    },
+    "httpd_access": {
```

Comment:
@nkanon Why we have 2 datasets in the schema

Reply:
Good catch Sherlock!